### DISCUSSION TITLE
Security headers

### DISCUSSION THREAD

**Discussion by khannzxc** [razonyang/hugo-theme-bootstrap#142] [Show and tell]
I have checked the site with https://securityheaders.com/

Found the below issues.

![sec11](https://user-images.githubusercontent.com/40522970/127012455-bb9aefb0-fc07-4950-99e4-908245d2b7bb.JPG)

Find some code and added in netlify.toml
```
[[headers]]
  for = "/*"
  [headers.values]
    SameSite = "Lax"
    Referrer-Policy = "strict-origin-when-cross-origin"
    X-Frame-Options = "SAMEORIGIN"
    X-XSS-Protection = "0"
    X-Content-Type-Options = "nosniff"
    Content-Security-Policy = "	default-src * data: blob: 'self';script-src https://utteranc.es/ *.github.com *.netlify.com *.facebook.com *.fbcdn.net *.facebook.net *.google-analytics.com *.google.com 127.0.0.1:* 'unsafe-inline' 'unsafe-eval' blob: data: 'self';style-src data: blob: 'unsafe-inline' *;connect-src *.facebook.com facebook.com *.fbcdn.net *.facebook.net wss://*.facebook.com:* wss://*.whatsapp.com:* attachment.fbsbx.com ws://localhost:* blob: *.cdninstagram.com 'self';block-all-mixed-content;upgrade-insecure-requests;"
```

And all the tests get passed.
![sec22](https://user-images.githubusercontent.com/40522970/127013027-de9ed910-6d2b-4fea-bf42-e6768ca76da0.JPG)

I think it has some effects on site ranking. what are your guys' thoughts?

**Comment by razonyang** [comment]
Thanks for pointing it out. I think it would be good to add those parameters into **netlify.toml**, so that the **netlify.toml** can be a good example for users who host their sites on Netlify.
